ui/packages/consul-ui/app/components/hashicorp-consul/index.hbs

```diff
@@ -261,6 +282,14 @@
                       </button>
                     </BlockSlot>
                   </ModalDialog>
+                  <Portal @target="app-before-skip-links">
```

Comment:
I dunno, when I built it I kinda felt that the login/logout button should potentially go before the Skip Main Nav link, so I made a `before/after` slot instead of just th3e single after slot. I vaguely remember considering putting the Login in the before slot and the Logout in the after slot - but I must have decided against that - I think it could have been due to not wanting to move the Login/Logout button around depending on whether you are logged in or out.

All in all, I'm fine with where ever it goes. Where do you think it should go? Before or after the Skip to Main Nav link?

Reply:
🤔 both as before-skip makes sense now.